src/main.rs

```diff
@@ -57,13 +64,13 @@ fn index() -> io::Result<NamedFile> {
 }
 
 #[get("/tables")]
-fn list_tables(client: State<DynamoDbClient>) -> String {
+fn list_tables(client: State<DynamoDbClient>) -> Result<Json<ScanOutput>, status::NotFound<String>>  {
```

Comment:
Looking at this again I guess our immediate problem here is the fact `rocket_contrib::Json` is of type `rocket::respond::Response` itself. 

We may achieve results when serialalizing `ScanOutput` ourselves. I think we'll need to implement `Serde::Serialize` for `ScanOutput` for this to work - we might me better of with learning how to ser/de on smaller structs first.

Reply:
I did a quick test an we should create or own repsonse object and not reuse the ScanOutput
Something like this works:

```
fn list_tables(client: State<DynamoDbClient>) -> Result<Json<Employee>, status::NotFound<String>>  {
    let mut scan_input = ScanInput::default();
    scan_input.table_name = String::from("rust-skillgroup");

    match client.scan(&scan_input).sync() {
        Ok(scan_output) => Ok(Json(Employee {id: String::from("id"), name: String::from("name")})),
        Err(scan_error) => Err(status::NotFound("Leg dich gehackt!".to_string())),
    }
}
```

The list_tables method was just a test anyway. So I would just create a get_employees method to get a list of all employees :)